lapis2/src/main/kotlin/org/genspectrum/lapis/logging/RequestId.kt

```diff
@@ -0,0 +1,43 @@
+package org.genspectrum.lapis.logging
+
+import jakarta.servlet.FilterChain
+import jakarta.servlet.http.HttpServletRequest
+import jakarta.servlet.http.HttpServletResponse
+import org.genspectrum.lapis.openApi.REQUEST_ID_HEADER
+import org.slf4j.MDC
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+import org.springframework.web.context.annotation.RequestScope
+import org.springframework.web.filter.OncePerRequestFilter
+import java.util.UUID
+
+@Component
+@RequestScope
+class RequestIdContext {
+    var requestId: String? = null
+}
+
+private const val REQUEST_ID_MDC_KEY = "RequestId"
+private const val HIGH_PRECEDENCE_BUT_LOW_ENOUGH_TO_HAVE_REQUEST_SCOPE_AVAILABLE = -100
```

Comment:
Is there docu we could link?

Reply:
Not really. This was try and error. `-500` doesn't work.